annif/corpus/subject.py

```diff
@@ -11,6 +11,9 @@
 from .skos import serialize_subjects_to_skos
 from .types import Subject, SubjectCorpus
 
+logger = logger.getChild("subject")
```

Comment:
Could we avoid reusing the same variable `logger` here? E.g.

    logger = annif.logger.getChild("subject")